packages/tool-cache/src/tool-cache.ts

```diff
@@ -477,7 +483,7 @@ export async function cacheFile(
   // anti-virus software having an open handle on a file.
   const destPath: string = path.join(destFolder, targetFile)
   core.debug(`destination file ${destPath}`)
-  await io.cp(sourceFile, destPath)
+  await io.cp(sourceFile, destPath, {preserveTimestamps: preserveTimestamps})
```

Comment:
Having `{x: x}` is equivalent to `{x}`:
```suggestion
  await io.cp(sourceFile, destPath, {preserveTimestamps})
```

Reply:
TIL.